includes/class-wc-product-variation.php

```diff
@@ -583,21 +583,4 @@ protected function get_valid_tax_classes() {
 
 		return $valid_classes;
 	}
-
-	/**
-	 * Delete variation, set the ID to 0, and return result.
-	 *
-	 * @since  4.4.0
-	 * @param  bool $force_delete Should the variation be deleted permanently.
-	 * @return bool result
-	 */
-	public function delete( $force_delete = false ) {
```

Comment:
If we remove the `delete` method, wouldn't that break backwards compatibility?

Reply:
This method exists already in the base class (`abstract-wc-product.php`). I removed the method here because it's useless (it's just calling the method in the base class).